### ISSUE TITLE
Add HBO Platinum sponsor!

### ISSUE THREAD

**Issue by rmw** [WriteSpeakCode/writespeakcode.github.io#22]
I've forward it your their logo!

In addition to adding to the sponsors.yml file, we'll need to add to the bottom of all pages of the website.

Are you ready to dive into [middleman](http://middlemanapp.com)?? :confetti_ball: 

Take a look at the `source/layouts/base_layout.haml` file.  Also look at the `source/_sponsor_footer.html` file.

Basically, we'll want to add something like:
"Write/Speak/Code is only possible through the generous support of our [amazing sponsors](/conference/sponsors):"

Then have a ul that loops through the platinum sponsors and puts their logo in an li and img tag.  Take a look at the `conference/sponsors.html.haml` file and then look at the CSS for the `bronze` sponsors - we are using `flexbox` for the display of the logos.  You are going to learn some awesome stuff!!

Email me if you have questions.

**Comment by dalrayes** [comment]
Hey Rebeca,

Bit confused on this one. When I added the blurb 'WSC would not be
possible..." to source/layouts/base_layout.haml, it added it to all the
pages. Its ok on the index page, but on the speakers page, for example,
there is already a sentence about that in place. Which one should I keep?
Or am I added it in the wrong place.

I currently have it in line 93 (before the %footer.footer)

I haven't added the logo loop yet, but wanted to check on place first.

thanks,

Dalal
